sdk/storage/storage-queue/src/generated/lib/models/index.ts

```diff
@@ -1,1372 +1,1051 @@
- * @extends RequestOptionsBase
  */
-export interface ServiceListQueuesSegmentOptionalParams extends msRest.RequestOptionsBase {
+export interface ServiceListQueuesSegmentOptionalParams extends coreHttp.RequestOptionsBase {
   /**
-   * @member {string} [prefix] Filters the results to return only queues whose
-   * name begins with the specified prefix.
+   * Filters the results to return only queues whose name begins with the specified prefix.
    */
   prefix?: string;
   /**
-   * @member {string} [marker] A string value that identifies the portion of
-   * the list of queues to be returned with the next listing operation. The
-   * operation returns the NextMarker value within the response body if the
-   * listing operation did not return all queues remaining to be listed with
-   * the current page. The NextMarker value can be used as the value for the
-   * marker parameter in a subsequent call to request the next page of list
-   * items. The marker value is opaque to the client.
+   * A string value that identifies the portion of the list of queues to be returned with the next
+   * listing operation. The operation returns the NextMarker value within the response body if the
+   * listing operation did not return all queues remaining to be listed with the current page. The
+   * NextMarker value can be used as the value for the marker parameter in a subsequent call to
+   * request the next page of list items. The marker value is opaque to the client.
    */
   marker?: string;
   /**
-   * @member {number} [maxresults] Specifies the maximum number of queues to
-   * return. If the request does not specify maxresults, or specifies a value
-   * greater than 5000, the server will return up to 5000 items. Note that if
-   * the listing operation crosses a partition boundary, then the service will
-   * return a continuation token for retrieving the remainder of the results.
-   * For this reason, it is possible that the service will return fewer results
-   * than specified by maxresults, or than the default of 5000.
+   * Specifies the maximum number of queues to return. If the request does not specify maxresults,
+   * or specifies a value greater than 5000, the server will return up to 5000 items. Note that if
+   * the listing operation crosses a partition boundary, then the service will return a
+   * continuation token for retrieving the remainder of the results. For this reason, it is
+   * possible that the service will return fewer results than specified by maxresults, or than the
+   * default of 5000.
    */
   maxresults?: number;
   /**
-   * @member {ListQueuesIncludeType} [include] Include this parameter to
-   * specify that the queues's metadata be returned as part of the response
-   * body. Possible values include: 'metadata'
+   * Include this parameter to specify that the queues's metadata be returned as part of the
+   * response body.
    */
-  include?: ListQueuesIncludeType;
+  include?: ListQueuesIncludeType[];
```

Comment:
This looks breaking too.  I wonder whether the original generated code is wrong, or intentionally changed to a singleton, as the only valid type here is `'metadata'`.  @XiaoningLiu do you remember anything?

Reply:
Looks like it's represented as `array` in the [OpenAPI json](https://msazure.visualstudio.com/One/_git/Storage-XStore?path=%2Fsrc%2FXFE%2FOpenApi%2FMicrosoft.QueueStorage%2Fpreview%2F2018-03-28%2Fqueue.json&version=GBmaster&line=1711&lineStyle=plain&lineEnd=1712&lineStartColumn=1&lineEndColumn=1):

```json
    "ListQueuesInclude": {
      "name": "include", 
      "in": "query", 
      "required": false, 
      "type": "array",
      "collectionFormat": "csv",
      "items": {
      "type": "string", 
      "enum": [ 
        "metadata" 
      ], 
      "x-ms-enum": { 
        "name": "ListQueuesIncludeType", 
        "modelAsString": false 
        }
      },
      "x-ms-parameter-location": "method",
      "description": "Include this parameter to specify that the queues's metadata be returned as part of the response body."
    },
```

Reply:
I believe this is a swagger breaking change introduced in https://msazure.visualstudio.com/One/_git/Storage-XStore/commit/76530600659f4d591d65266c74b2d96c785c3015?_a=compare&path=%2Fsrc%2FXFE%2FOpenApi%2FMicrosoft.QueueStorage%2Fpreview%2F2018-03-28%2Fqueue.json 

We can take this change. But please update BreakingChange.md for this.